roles/openstack-meta/defaults/main.yml

```diff
@@ -439,6 +439,14 @@ openstack_meta:
           config_files: /etc/neutron/neutron.conf,/etc/neutron/services/loadbalancer/haproxy/lbaas_agent.ini,/etc/neutron/neutron_lbaas.conf
           restart: on-failure
           kill_mode: process
+      f5_oslbaasv2_agent:
```

Comment:
yes, you are right. when there is need for rhosp, we just need to add the few lines of code. Of course we also need to add more lines to use systemd service.